src/arrayMethodSort.js

```diff
@@ -4,9 +4,19 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function(compareFunction =
+  (item1, item2) => (item1 + '') > (item2 + '')) {
```

Comment:
Please move default compareFunction value to separate variable for better code readability